api/stats.py

```diff
@@ -0,0 +1,111 @@
+def load():
+	from pickle import load
+	stats_dict = {}
+	stats_db = open('api/stats.db', 'rb')
+	try:
+		stats_dict = load(stats_db)
+		return stats_dict
+	except:
```

Comment:
Only accept the exception that you're expecting. Otherwise this could catch KeyboardInterrupt or something which needs to be handled somewhere else.